invenio.cfg

```diff
@@ -562,3 +572,92 @@ VOCABULARIES_NAMES_SCHEMES = {
 # We override the templates to add new fields needed for the migrated statistic events
 _APP_RDM_STATS_EVENTS["file-download"]["templates"] = "cds_rdm.stats.templates.events.file_download"
 _APP_RDM_STATS_EVENTS["record-view"]["templates"] = "cds_rdm.stats.templates.events.record_view"
+
+STATS_EVENTS = {
```

Comment:
why do we need to dump the full config here?

Reply:
Thanks for the comment! I can simply override/add the params in the same way as done with the template and like that, there is no need to override the full conf ....